library/chemistry/src/JordanWigner/JordanWignerOptimizedBlockEncoding.qs

```diff
@@ -54,20 +55,16 @@ struct OptimizedBEGeneratorSystem {
 
 // Get OptimizedBETermIndex coefficients
 function GetOptimizedBETermIndexCoeff(term : OptimizedBETermIndex) : Double {
-    let (a, b, c, d, e, f) = term!;
-    return a;
+    term.Coefficient
 }
```

Comment:
For the two places this is used you could just instead add `.Coefficient` to whatever argument was being passed, and then eliminate this function.

Reply:
Updated.